File: Go/Surface_area_and_volume_of_box.go
// Write a function that returns the total surface area and volume of a box as an array: [area, volume]

package main

import "fmt"

var l int = 2

func main() {
	result := boxCalulations()
	fmt.Println("surface area of box is ", area)
	fmt.Println("surface area of box is ", volume)

}

func boxCalulations() []int {
	area = 6 * (l * 2)
	volume = l * l * l
	return []int{area, volume}

}

func boxCalculations() []int {
	area := 6 * (l * 2)
	volume := l * l * l
	return []int{area, volume}
}

File: JavaScript/8_Kyu/Surface_area_and_volume_of_box.js
/*
Write a function that returns the total surface area and volume of a box.

The given input will be three positive non-zero integers: width, height, and depth.

The output will be language dependant, so please check sample tests for the corresponding data type, (list, tuple, struct, query, et cetera).

*/


function getSize(width, height, depth) {
    const totalSurfaceArea = 2 * ((depth * width) + (depth * height) + (width * height))
    const totalVolume = width * height * depth;

    return [totalSurfaceArea, totalVolume]
}